voter/models.py

```diff
@@ -102,6 +103,7 @@ class NCVoter(models.Model):
     class Meta:
         verbose_name = "NC Voter"
         verbose_name_plural = "NC Voters"
+        # db_table = 'voter_ncvoter'
```

Comment:
Does this mean you've tested it and it works without it? If so, can we remove the comment?